editoast/src/views/layers/info.rs

```diff
@@ -0,0 +1,31 @@
+use crate::api_error::ApiResult;
+use crate::chartos::LayersDescription;
+use rocket::serde::json::{json, Value as JsonValue};
+
+use rocket::State;
+
+#[get("/info")]
+pub async fn info_route(layers_description: &State<LayersDescription>) -> ApiResult<JsonValue> {
+    Ok(json!(layers_description.layers))
+}
+
+#[cfg(test)]
+mod tests {
+    use crate::views::tests::create_test_client;
+    use rocket::http::Status;
+    use serde_json::Value;
+    use std::fs::File;
+    use std::io::BufReader;
+
+    #[test]
+    fn info() {
+        let client = create_test_client();
+        let response = client.get("/layers/info").dispatch();
+        assert_eq!(response.status(), Status::Ok);
+        let expected_body: Value = serde_json::from_reader(BufReader::new(
+            File::open("./tests/data/info.json").unwrap(),
+        ))
+        .unwrap();
+        assert_eq!(response.into_string().unwrap(), expected_body.to_string());
```

Comment:
I'm don't like this test because each time we are going to update the config file, we need to update the produced test file.
We can do lighter check on the resulted payload or just check that the endpoint returns a 200.

Reply:
I'm not sure to see why it is an issue to change the content of the test file if indeed the response has changed (it will be easy to spot when reviewing the change in the future). But if you really want me to drop this comprehensive test then I drop it

Reply:
The issue here is that without changing the endpoint behavior, this test will fail when we change the chartos config file. 

Ideally we could configure the chartos config file. We could give a custom configuration to the test server and test the response manually.  The drawback is that we cannot read the config file at compile time anymore.

Reply:
With what we said this morning, a better tests would be to check that generated sql queries thanks to the config file are correct and indeed testing content of the config file will be inadequate, I'm working on this and I will ping you once I'm done :)